logstash-core/lib/logstash/agent.rb

```diff
@@ -22,7 +22,7 @@ class LogStash::Agent
 
   def initialize(options = {})
     @pipelines = {}
-     
+
```

Comment:
Gif diff are weird sometime, you know you can pass `?w=1` to the PR url,  this will make the diff ignore all the whitespace changes. The downside you cannot comment on the pr in that context... We can't have all the nice things.